backend/src/models/postQuestionsList.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "question": "What topics did you discuss?",
+    "type": "bullet",
+    "id": 0
```

Comment:
Let's remove the id from here because currently id should be a hash which won't change until the question/type is changed